framework/assets/yii.validation.js

```diff
@@ -36,7 +36,7 @@ yii.validation = (function ($) {
             }
         },
 
-        'boolean': function (value, messages, options) {
+        boolean: function (value, messages, options) {
```

Comment:
This is because in es3 boolean is a keyword, and could not be used without quotes.
It's ok in es5.